src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -204,4 +208,187 @@ public int draw() {
         }
         return normalizedHistogram;
     }
+
+    /**
+     * Determines if a variant would be called under the particular conditions of a given total depth, alt depth,
+     * average base qualities, allele fraction of variant and log odds threshold necessary to call variant.
+     * @param totalDepth Depth at the site to be called, both alt and ref.
+     * @param altDepth Number of alt bases at this site.
+     * @param sumOfAltQualities Average Phred-scaled quality of bases
+     * @param alleleFraction Allele fraction we are attempting to detect
+     * @param logOddsThreshold Log odds threshold necessary for variant to be called
+     * @return Whether or not the model would call a variant given the parameters
+     */
+     static boolean isCalled(final int totalDepth, final int altDepth, final double sumOfAltQualities, final double alleleFraction, final double logOddsThreshold) {
```

Comment:
`public`?

Reply:
Oh, in that case does Picard have a dependency with the `@VisibleForTesting` annotation?